next-packages/core/src/keyframes.js

```diff
@@ -1,15 +1,23 @@
 // @flow
-import type { ScopedInsertableStyles } from '@emotion/utils'
+import css from '@emotion/css'
 
 type Keyframes = {|
   name: string,
-  styles: string
-|}
+  styles: string,
+  anim: 1,
+  toString: () => string
+|} & string
```

Comment:
how does it work 😨 ? I mean intersection with string

Reply:
It’s not really good at all and there has to be a FlowFixMe comment but it’s like this so people can interpolate it in a string without flow errors